sdks/python/apache_beam/runners/portability/stager_test.py

```diff
@@ -242,7 +242,7 @@ def test_with_main_session(self):
 
   # (BEAM-13769): Remove the decorator once cloudpickle is default pickle
   # library
-  @pytest.mark.skip
+  @pytest.mark.no_xdist
```

Comment:
I'm just curious about where we access these markers in our test rules.

Reply:
sometimes i have to grep but in this case github's search shows relevant results: https://github.com/apache/beam/search?q=no_xdist